### PR TITLE
Solving bug of error resolution "ul ol+li{fill:#000000;font-family:ZW…

### PR DESCRIPTION
**Pull request by mccj**
#### What does this implement/fix? Explain your changes.

Solving bug of error resolution "ul ol+li{fill:#000000;font-family:ZWBIQX+HelveticaUnicodeMS;}"

### CODE REVIEW

**Comment by mccj** [comment]
Converting PDF to SVG file with FreeSpire.PDF generates the following code
`<text style="fill:#000000;font-family:AAAAAA+WenQuanYiMicroHei-0;" font-size="7" x="132.8691" y="22.13528">B</text>`
Style symbol `+` cannot be parsed at present.

**Comment by mrbean-bremen** [comment]
As far as I can see, this looks good - thanks!
Though I would wait a bit until #563 is resolved for ExCSS.

File: Source/Svg.csproj [hunk 1]
```diff
@@ -152,7 +152,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Fizzler" Version="1.1.0" />
+    <PackageReference Include="Fizzler" Version="1.1.0" Condition="'$(TargetFramework)'=='net35' Or '$(TargetFramework)'=='net452'"/>
```
Comment (H1Gdev) [inline Source/Svg.csproj]: Is this change necessary ?
ref. https://github.com/vvvv/SVG/issues/563#issuecomment-524795988

File: Tests/Svg.UnitTests/ExCSSLexerTests.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    ///to contain all CssQueryTest Unit Tests
+    ///</summary>
+    [TestFixture]
+    public class ExCSSLexerTests
+    {
+        [Test]
+        public void LexerStylesheet()
+        {
+            //
+            TestStylesheetReader("#a{fill:#000000;font-family:ZWBIQX+HelveticaUnicodeMS;}", 14);
```
Comment (H1Gdev) [inline Tests/Svg.UnitTests/ExCSSLexerTests.cs]: I think that check test results.

tokens is thefollowing.
```
font-family
:
ZWBIQX
+
HelveticaUnicodeMS
;
```
Is expected `font-family` value `ZWBIQX+HelveticaUnicodeMS` ?
In that case, I think that it is better to implement it with reference to `-`.

File: Tests/Svg.UnitTests/ExCSSLexerTests.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using System.Linq;
+
+namespace Svg.UnitTests
+{
+    /// <summary>
+    ///This is a test class for CssQueryTest and is intended
```
[the remark this answers is not in the document]
Comment (mccj) [reply Tests/Svg.UnitTests/ExCSSLexerTests.cs]: I'm sorry

File: Source/Svg.csproj (hunk 1 above)
Comment (mccj) [reply Source/Svg.csproj]: Not necessary, but Fizzler 1.1.0 is incompatible with NET Core 2

Comment (H1Gdev) [reply Source/Svg.csproj]: So, it should not be included in this PR.

File: Source/SvgElementIdManager.cs [hunk 4]
```diff
@@ -1,217 +1,204 @@
...
-        }
-
-    }
-
-    public class SvgElementEventArgs : EventArgs
-    {
-        public SvgElement Element;
-    }
-}
+﻿using System;
```
Comment (H1Gdev) [inline Source/SvgElementIdManager.cs]: It seems that the line feed code has changed.

File: Source/SvgDocument.cs [hunk 5]
```diff
@@ -74,6 +76,33 @@ private static int GetSystemDpi()
             }
             return _fontDefns;
         }
+        internal FontFamily[] PrivateFontDefns()
+        {
+            if (_privateFonts == null)
+            {
+                _privateFonts = new PrivateFontCollection();
```
Comment (H1Gdev) [inline Source/SvgDocument.cs]: There is [remarks](https://docs.microsoft.com/dotnet/api/system.drawing.text#remarks) in the API reference.
What do you think about this ?

**Comment by mrbean-bremen** [comment]
Superseded by #595 - closing.
